### PR TITLE
initial homePg component

### PR DESCRIPTION
**Pull request by aurobindoGupta**
this branch only contains the design and not the functionalities.
image

### CODE REVIEW

File: src/Components/navbar/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <div>
+      <nav className="navBar">
+        <div className="heroContainer">
+          <p className="heroTitle" onClick={() => navigate("/")}>
+            MyWebsite
+          </p>
+        </div>
+        <div className="navBar-searchBar">
+          <form className="searchBar">
+            <input className="searchBar-input" placeholder="Search" />
```
Comment (ckahub) [inline src/Components/navbar/NavBar.js:15]: use camel case everywhere defining class a

File: src/pageContainer/homePg/homePg.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+.baseContainer {
+    max-width: 100%;
+    background: var(--background-color);
+    overflow: scroll;
+  }
+  .pageContent {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    padding: 0.2rem 10.313rem;
```
Comment (ckahub) [inline src/pageContainer/homePg/homePg.css:13]: how u came with 10.313?

File: src/index.css [hunk 3]
```diff
@@ -1,13 +1,19 @@
 body {
   margin: 0;
+  height: 100%;
+  width: 100%;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
```
Comment (ckahub) [inline src/index.css:6]: too many font family

**Review by ckahub — COMMENTED** [review]
code looks neat and clean

**Review by ckahub — APPROVED** [review]
code looks neat and clean

File: src/pageContainer/homePg/homePg.css (hunk 2 above)
Comment (aurobindoGupta) [reply src/pageContainer/homePg/homePg.css:13]: 165px to rem is 10.313

**Review by Vivek-Py — APPROVED** [review]
One thing you can improve upon is class naming, follow a uniform pattern. You have combined `camelCase ` with `dash-case`. Having a uniform/universe pattern, this makes it easy for code review. Otherwise code looks clean, keep up!

**Review by MayurChawla — COMMENTED** [review]
thumbs up bro.. good work, keep it up

**Pull request merged**